backend/controllers/prisonerProfile/prisonerVisits.ts

```diff
@@ -157,8 +157,9 @@ export default ({ prisonApi, pageSize = 20 }) =>
         results,
         visitTypes: VISIT_TYPES,
         filterApplied: Boolean(fromDate || toDate || visitType),
-        prisons: hasLength(prisons) && prisons.map((type) => ({ value: type.prisonId, text: type.prison })),
```

Comment:
this was causing `false` to be sent to nunjucks which then didn't add the `All` option